static-site/src/components/ListResources/index.tsx

```diff
@@ -67,11 +74,17 @@ function ListResources({
   return (
     <ListResourcesContainer>
 
-      <Showcase cards={showcaseCards} setSelectedFilterOptions={setSelectedFilterOptions}/>
+      { showcaseCards.length > 0 && (
+        <Showcase cards={showcaseCards} setSelectedFilterOptions={setSelectedFilterOptions}/>
+      )}
 
       <Filter options={availableFilterOptions} selectedFilterOptions={selectedFilterOptions} setSelectedFilterOptions={setSelectedFilterOptions}/>
 
-      <SortOptions sortMethod={sortMethod} changeSortMethod={changeSortMethod}/>
+      { groups && groups[0]?.lastUpdated && (
```

Comment:
[not a request for changes - I just wanted to see if this was possible and am noting it here]

So you can use the elvis operator for array indexing too! `groups?.[0]?.lastUpdated && ...`